### ISSUE TITLE
Qiqqa is down since weeks. I need your support extracting or sync my web library. Constant crashes are also an issue. Unable to find any email id of support contact

### ISSUE THREAD

**Issue by arunendramishra** [jimmejardine/qiqqa-open-source#358]
### Discussed in https://github.com/jimmejardine/qiqqa-open-source/discussions/344

<div type='discussions-op-text'>

<sup>Originally posted by **Waxwood** July 15, 2021</sup>
I have been experimenting with Qiqqa for only 2 days and so far, it appears to be an amazing tool. I have tried to keep up with innovative software tools since the early 2000s yet I have no idea why I did not try Qiqqa when I was writing my PhD a few years ago.
Qiqqa's' annotation features allowing the user to highlight text, select sections of text or photos on a PDF and add a keyword to it are amazingly useful and allow it to be used as a simple QDA software tool. I have not seen any other low-cost tool that can do this (I assume how ever that Citavi can do this). 

My only concern is that Qiqqa is not cross-platform and will not run in Linux.
I abandoned Windows years ago and only use it when I must. 
For now, I am running Qiqqa in a virtual Windows 10, but have encountered ongoing crashes that prevent me from seriously using the software. 
Please tell me that a cross-platform version will be released in the near future!!

Thank you for open sourcing Qiqqa- I hope you will be able to build a strong development team. Its an amazing application that more students and researchers should be using.

A message to researchers and graduate students: You really must check out Qiqqa. It's probably the most powerful research tool available (unless you use a pro-level QDA application costing between $400 and $1500 UDS), and it is open source which means that a company can never deny you access to your research project.</div>

**Comment by arunendramishra** [comment]
> Hi, I have the same trouble...Did you find a solution?

Hi , sorry but couldn't find anything till now. Unable to contact anyone from Qiqqa and surprisingly no email or support link provided. It's a loss of almost 4 years work for  me.

**Comment by GerHobbelt** [comment]
Re Qiqqa.com and web libraries: also note the blurb here: https://github.com/jimmejardine/qiqqa-open-source#warning-notice-for-commercial-qiqqa-users-with-a-user-account-and-web-libraries 

So I guess qiqqa.com (the original commercial Qiqqa site) is finally down 🤔 (Quantisle announced back in 2019 that this would happen somewhere around Q4 2020, so we've had an extra year.

## But that doesn't help resolve the actual problems. So here's how to approach this:

First off, Qiqqa has always kept a **local copy of your libraries** so my first question for *anyone* with a library issue is always:

**Do you still have access to the machine you've been running Qiqqa on?**

If the answer to that one is `YES`, than we can 'recover' (or rather: continue to use your libraries). More on that below.

If the answer to that is `NO`, than things will be *quite hairy* as any 'recovery' of Qiqqa databases *starts* with having access to the local harddisk in your machine where you have installed and been using Qiqqa on, even if only using that machine *some* of the time, e.g. when you've been using Commercial Qiqqa in a both-at-home-and-at-the-office(s) type of scenario. 

> ^^^ Anyone who's in the latter category, will have some tough time. We *may* be able to help, but all positive results start with getting (or having) access to any of the actual Qiqqa machines. 
>

### That being said, I will assume you still have access to your local Qiqqa machine you've been using before.

For the risk-averse, it's a two-stage process:

1. create a **full backup of the current library collection** to ensure nothing can get lost.
2. install a new Qiqqa version and have it (auto-)discover your library set.

## 1. create a **full backup of the current library collection** to ensure nothing can get lost.

Re stage 1: all Qiqqa user data is stored in a directory set inside the so-called "**Qiqqa base directory**": you can find out where that directory is located in your personal machine by following the guide here: https://github.com/jimmejardine/qiqqa-open-source/blob/master/docs-src/Notes/FAQ/How%20to%20locate%20your%20Qiqqa%20Base%20Directory.md

**Do note that the last bit of that guide addresses so extra work that's *irrelevant here*, as it was originally written for someone who had a specific issue with one particular library in their Qiqqa library set, so ignore the last couple of paragraphs in there that go on about inspecting the subdirectories and config files, hunting for a specific library.**

**What *you* need to do instead is BACKUP EVERYTHING in the QIQQA BASE DIRECTORY.** (ZIPping it ([or using other archival tools](https://www.7-zip.org/)) and storing that ZIP archive elsewhere would be a fine idea.)


## 2. install a new Qiqqa version and have it (auto-)discover your library set.

All Qiqqa v8x (v80, v81, v82, v83, ...) releases are *binary compatible* and can be installed *over* one another, *replacing* the existing install. These are also binary compatible with the last Commercial Qiqqa release (v79), so you can safely install an open-source Qiqqa '*over*' the current Qiqqa.

While Open Source Qiqqa **does not support Qiqqa Cloud access**, you can still access and use those libraries locally as we **can access** that local copy that Qiqqa has been using all the time anyway. (The Cloud Access code and keys were never open-sourced by Quantisle, so even if that site were still up, I wouldn't bee able to add that functionality back into Qiqqa again.)

TL;DR: open-source Qiqqa doesn't ask for a log-in code, **does** auto-search and import all existing Qiqqa libraries it can find, **including your original Qiqqa Cloud libraries** as it will inspect their local copies, which *should be up-to-date* if the machine you're using *now* has been the machine you used Commercial Qiqqa on *last*. (Otherwise your local copy will only carry the material state of affairs at the time of your last Sync action in this local Qiqqa.)

For a bit of extra help with installing Qiqqa (and places to get the various releases), see https://github.com/jimmejardine/qiqqa-open-source/blob/master/docs-src/Notes/FAQ/Installing%20Qiqqa%20-%20Updating%20Qiqqa.md

## Additional, important bit of advice

When you're going to install Qiqqa, please use the latest v82 release first, as that is the most stable, generally. The v83 releases are okay for some people, but are all created as test releases for particular problems and have some known bugs, *which may bite you*.

### What if I don't like/want v82?

If you don't like it at all, then you could opt for the closest possible equivalent of the v79 version, which is v80, the first (and last) OSS release from Quantisle / Jardine -- which is basically v79 with all Cloud Access stripped off plus a couple of quick tweaks to make the beast run without commercial license info & validation. 

> I don't recommend using v80 as it shared the same bugs  with v79, which killed my own libraries and Qiqqa usage, but YMMV -- if you were happy using v79, you're probably okay with v80 as well.
>
> However, **do realize** that v80 has some bugs around thee library discovery code, so **my advice** is to use v82. If you do not like it, you can always backpedal by installing v80, which will replace/overwrite it as **any** Qiqqa version will install over older or newer previously installed releases. Ergo: you can try any version you like from the list of releases and always go back to a version you liked better.

----

Let me know how it goes in the comments below. 👍

**Comment by arunendramishra** [comment]
Thanks @GerHobbelt for such a detailed answer, I really appreciate. But the issue is my Windows crashed and I lost everything on my local machine and no backup at all. I regularly sync my work on my web library and that contains all my 850+ papers / pdfs. It's all my 4 years work. Any how; can I get the access to my web library and download/recover it. I am sorry, was not aware about Qiqqa's exit news.
